packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -142,6 +142,21 @@ export class GCTelemetryTracker {
 
 		const nodeStateTracker = this.getNodeStateTracker(nodeUsageProps.id);
 		const nodeType = this.getNodeType(nodeUsageProps.id);
+		const timeout = (() => {
+			switch (nodeStateTracker?.state) {
+				case UnreferencedState.Inactive:
+					return this.configs.inactiveTimeoutMs;
+				case UnreferencedState.TombstoneReady:
+					return this.configs.sweepTimeoutMs;
+				case UnreferencedState.SweepReady:
+					return (
+						this.configs.sweepTimeoutMs &&
+						this.configs.sweepTimeoutMs + this.configs.tombstoneSweepDelayMs
```

Comment:
@agarwal-navin what do you think of this logic - for SweepReady logs, `timeout` will be the "total" sweep timeout since unreferenced transition.

Alternative would be to just do the delay here.  In the (maybe near?) future when we wait to start the delay until after a node has been Tombstoned, I would switch it to the delay here.  I think it's best to do it this way for now and change it later.

Reply:
`timeout` is just the configured timeout. So, it should be tombstone timeout + sweep delay.
`age` is how long this node has been unreferenced for and I think that is what you mean?

Reply:
I think we're saying the same thing.  If this looks good to you go ahead and resolve the conversation.